Assembly-CSharp/Console.cs

```diff
@@ -85,7 +121,7 @@ public void Start()
         [PublicAPI]
         public void Update()
         {
-            if (!Input.GetKeyDown(KeyCode.F10))
+            if (!Input.GetKeyDown(_toggleKey))
```

Comment:
Is there a good reason to use string over keycode?

Reply:
I think `f10` in config file is better than `289`